### ISSUE TITLE
The training results seem not to converge

### ISSUE THREAD

**Issue by dengxy180** [Leeiieeo/AG-Pose#15]
Due to the CyclicLR, the test results of the training weights within each epoch show considerable differences. Even after changing the learning rate strategy, this issue persists, making it difficult to compare because the reproduced results seem not converge. Could you provide detailed information about evaluation in paper? For example, how many total epochs were trained, whether the weights were taken from the final result or the best result? And other relevant details.
